<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonInforDialog</class>
 <widget class="QDialog" name="PersonInforDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_bkg{
    background-color:#D3D3D3;
    border-radius: 4px;
	border:1px solid #000000;
	
	
}
QWidget
{
	font-size:12px;
    color:#333333;
	font: 10pt &quot;微软雅黑&quot;;
}
QLabel{
    font-size:12px;
    color:#333333;
	font: 10pt &quot;微软雅黑&quot;;
}

QWidget#widget
{
	background-color:#EAEAEA;
}

QLabel#title_label{
    font: bold 14px;
}

QLabel#sp_1,#sp_2,#sp_3,#sp_4,#sp_5,#sp_6,#sp_7,#sp_8,#sp_9{
    border:1px solid #D6D6D6;
}
QLabel#label_3,#label_12,#label_11,#label_SaveResult
{
	color:#FF4500;
}

QPushButton#pbCancle,#pbSave
{
    border-radius: 2px;
    border:1px solid #EBEBEB;
    background-color:#FFFFFF;
}

QPushButton#pbCancle:hover,#pbSave:hover{
    border-radius: 2px;
    border:1px solid #EBEBEB;
    background-color:#EBEBEB;
}


QPushButton#pbCancle:pressed,#pbSave:pressed{
    border-radius: 2px;
    border:1px solid #D6D6D6;
    background-color:#D6D6D6;
}
QPushButton#pbclose{
	border-top-right-radius: 4px;
    border:1px solid #00000000;
    background-color:#FFFFFF;
}

QPushButton#pbclose:hover{
    background-color:#FF0000;
}
QPushButton#pbclose:pressed{
    background-color:#A30707;
}
QScrollArea#scrollArea{
	border:none;
}

QWidget#scrollAreaWidgetContents{
	background-color:#FFFFFFFF;
}
QWidget#vline{
	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,  
                                 stop: 0 #00000000, stop: 0.1 #A3A6A9,  
                                 stop: 0.9 #A3A6A9, stop: 1.0 #00000000); 
}


QLineEdit
{
border:1px solid #D6D6D6;
    background-color:#FFFFFF;
}
QLineEdit:hover{
    border-radius: 2px;
    border:1px solid #EBEBEB;
    background-color:#EBEBEB;
}


QLineEdit:focus
{
    border:1px solid #4F9AD4;
    background-color:#FFFFFF;
}

QLabel#label_SaveResult
{
	font-size:18px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_bkg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="TitleBar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="title_label">
           <property name="text">
            <string>基本信息设置</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbclose">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="vline" native="true"/>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>496</width>
          <height>458</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>479</width>
           <height>563</height>
          </rect>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>481</width>
            <height>561</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>最大同时接待数:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="MaxNumEdit">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>17</y>
             <width>191</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>40</y>
             <width>201</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>*在线空闲时同时接待访客最大数。</string>
           </property>
          </widget>
          <widget class="QLabel" name="sp_1">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>70</y>
             <width>500</width>
             <height>1</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>90</y>
             <width>110</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>手机号:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="realNameEdit">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>190</y>
             <width>21</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="uin_label">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>90</y>
             <width>151</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>140</y>
             <width>110</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>登录名:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="Loginname_label">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>130</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>190</y>
             <width>110</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>真实姓名:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>270</y>
             <width>110</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>网页显示名:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="WebNameEdit">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>270</y>
             <width>20</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>290</y>
             <width>361</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>*对访客显示在线坐席修改后及时生效/离线坐席修改后24小时生效。</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>210</y>
             <width>301</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>*不超过25个字符（汉字，数字，字母和下划线）。</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>340</y>
             <width>110</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>所在部门:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="DepatrM_label">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>340</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
          <widget class="QLabel" name="sp_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>120</y>
             <width>541</width>
             <height>1</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="sp_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>170</y>
             <width>541</width>
             <height>1</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="sp_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>250</y>
             <width>500</width>
             <height>1</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="sp_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>320</y>
             <width>500</width>
             <height>1</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="sp_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>380</y>
             <width>500</width>
             <height>1</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="pbSave">
           <property name="geometry">
            <rect>
             <x>384</x>
             <y>510</y>
             <width>70</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pbCancle">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>510</y>
             <width>70</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>放弃</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_SaveResult">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>510</y>
             <width>241</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="Image_label">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>390</y>
             <width>181</width>
             <height>111</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="realNameLabel">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>190</y>
             <width>191</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
          <widget class="QLabel" name="WebNameLabel">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>270</y>
             <width>201</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
